frontend/spec/features/resources_spec.rb

```diff
@@ -0,0 +1,908 @@
+    archival_object = create(:archival_object, title: "Archival Object Title #{now}", resource: { 'ref' => resource.uri })
+    run_index_round
+
+    visit "resources/#{resource.id}/edit#tree::archival_object_#{archival_object.id}"
+
+    within '#form_archival_object' do
+      click_on 'Add Subject'
+    end
+
+    find('#archival_object_subjects_ .linker-wrapper a.btn').click
+    find('#archival_object_subjects_ a.linker-create-btn').click
+
+    within '#archival_object_subjects__0__ref__modal' do
+      find('button', text: 'Add Term/Subdivision', match: :first).click
+      select 'Local', from: 'subject_source_'
+      fill_in 'subject_terms__0__term_', with: "Test Term 1 #{now}"
+      select 'Function', from: 'subject_terms__0__term_type_'
+      fill_in 'subject_terms__1__term_', with: "Test Term 2 #{now}"
+      select 'Genre / Form', from: 'subject_terms__1__term_type_'
+
+      click_on 'Create and Link'
+    end
+
+    run_index_round
+
+    within '#archival_object_subjects__0__ref__combobox' do
+      find('.token-input-delete-token').click
+    end
+
+    fill_in 'token-input-archival_object_subjects__0__ref_', with: "Test Term 1 #{now}"
+    find('li.token-input-dropdown-item2').click
+
+    # Click on save
+    find('button', text: 'Save Archival Object', match: :first).click
+
+    expect(page).to have_text "Archival Object Archival Object Title #{now} updated"
+
+    element = find('#archival_object_subjects_ ul.token-input-list')
+    expect(element).to have_text "Test Term 1 #{now}"
+  end
+
+  it 'can view a read only Archival Object' do
+    now = Time.now.to_i
+    resource = create(:resource, title: "Resource Title #{now}")
+    archival_object = create(:archival_object, title: "Archival Object Title #{now}", resource: { 'ref' => resource.uri })
+    run_index_round
+
+    visit "resources/#{resource.id}/edit#tree::archival_object_#{archival_object.id}"
+
+    click_on 'Close Record'
+
+    element = find('.record-pane h2')
+    expect(element).to have_text archival_object.title
+  end
+
+  it 'exports and downloads the resource to xml' do
+    now = Time.now.to_i
+    resource = create(:resource, title: "Resource Title #{now}")
+    run_index_round
+
+    visit "resources/#{resource.id}"
+
+    files = Dir.glob(File.join(Dir.tmpdir, '*_ead.xml'))
+    files.each do |file|
+      File.delete file
+    end
+
+    click_on 'Export'
+    click_on 'Download EAD'
+
+    files = Dir.glob(File.join(Dir.tmpdir, '*_ead.xml'))
+    expect(files.length).to eq 1
+    file = File.read(files[0])
+    expect(file).to include(resource.title)
+  end
+
+  # NOTE: There is not link with text 'Print Resource to PDF',
+  #       inside export menu, on resouce page.
+  xit 'displays a link for downloading pdf' do
+    now = Time.now.to_i
+    resource = create(:resource, title: "Resource Title #{now}")
+    run_index_round
+
+    visit "resources/#{resource.id}"
+
+    click_on 'Export'
+    click_on 'Print Resource to PDF'
+  end
```

Comment:
@thimios maybe lets just delete this ignored spec since the pdf generation is correctly handled in the [frontend jobs feature spec](https://github.com/archivesspace/archivesspace/blob/master/frontend/spec/features/jobs_spec.rb#L53)

Reply:
good point